helpers.py

```diff
@@ -107,10 +100,10 @@ def log_user_in():
     # type in the wrong username/password.
     print(decision_prompt)
     while True:
-        decision = input("type here:")
+        decision = input("Type here: ")
         if decision == '1':
             # Get username
-            username = input("Email:")
+            username = input("Email: ")
```

Comment:
Does this extra space eventually get stripped when the login request is made? So that the db doesn't say wrong email just because there is a space in the input?

Reply:
I looked online and tested, and an extra space isn’t being added to the input value. I am able to log in successfully as well.

Reply:
Nice Nice!